hack/local-up-cluster.sh

```diff
@@ -366,7 +366,28 @@ function start_apiserver {
     APISERVER_PID=$!
 
     # We created a kubeconfig that uses the apiserver.crt
-    cat <<EOF | sudo tee "${CERT_DIR}"/kubeconfig > /dev/null
+    if [[ -n "${ALLOW_ANY_TOKEN:-}" ]]; then
+        cat <<EOF | sudo tee "${CERT_DIR}"/kubeconfig > /dev/null
+apiVersion: v1
+kind: Config
+clusters:
+  - cluster:
+      certificate-authority: ${ROOT_CA_FILE}
+      server: https://${API_HOST}:${API_SECURE_PORT}/
+    name: local-up-cluster
+contexts:
+  - context:
+      cluster: local-up-cluster
+      user: kubelet
+    name: service-to-apiserver
+current-context: service-to-apiserver
+users:
+- name: kubelet
+  user:
+      token: admin/system:masters
+EOF
+  else
```

Comment:
what about something like #36134? don't more things than just the master use this kubeconfig? the way we start local-up-cluster with separately secured components could probably use some rethinking in 1.6